File: XBOOK/XBOOK.Data/ViewModels/CompanyProfileViewModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace XBOOK.Data.ViewModels
{
    public class CompanyProfileViewModel
    {
        public int Id { get; set; }
        public string companyName { get; set; }
        public string address { get; set; }
        public string city { get; set; }
        public string country { get; set; }
        public string zipCode { get; set; }
        public string currency { get; set; }
        public string dateFormat { get; set; }
        public string bizPhone { get; set; }
        public string mobilePhone { get; set; }
        public string directorName { get; set; }
        public string logoFilePath { get; set; }
        public string taxCode { get; set; }
    }
}

File: XBOOK/XBOOK.Web/ClientApp/src/app/pages/_shared/models/companyprofile/companyprofile-view.model.ts
import { BaseModel } from '../../../../shared/model/base.model';

export class CompanyprofileView extends BaseModel {
  companyName: string;
  address: string;
  city: number;
  country: number;
  zipCode: string;
  currency: string;
  dateFormat: string;
  bizPhone: number;
  bankAccount: string;
  mobilePhone: string;
  directorName: string;
  logoFilePath: string;
  taxCode: number;
  code: string;
}
export class CompanyprofileViewModel {
  clientID: number;
  clientName: string;
  address: string;
  note: string;
  outstanding: number;
  overdue: number;
  bankAccount: string;
}
